.gitignore

```diff
@@ -0,0 +1,3 @@
+node_modules
```

Comment:
We should use a gitignore template here for nodejs/express projects.

Let's say this one, seems pretty okay
https://gist.github.com/andreasonny83/b24e38b7772a3ea362d8e8d238d5a7bc